workflow/common/convert.go

```diff
@@ -90,6 +90,13 @@ func toWorkflow(cronWf wfv1.CronWorkflow, objectMeta metav1.ObjectMeta) *wfv1.Wo
 	}
 
 	wf.Labels[LabelKeyCronWorkflow] = cronWf.Name
+	if wf.Spec.WorkflowTemplateRef != nil {
```

Comment:
two things here:

1. could we re-use the `setWfTemplateLabel` func here?
2. is this strictly necessary? the `operator` logic should handle this as well, no?
    - though I get that this would set the label before creating the Workflow vs that one would be after

Reply:
removed as it is not necessary